matter/efr32/efr32mg24/BRD2601B/autogen/sl_ot_init.c

```diff
@@ -29,11 +29,17 @@
  ******************************************************************************/
 
 #include "sl_ot_init.h"
+// TODO: Remove the Power Manager include when OT does not add an EM1 req at init
+#define CURRENT_MODULE_NAME "OPENTHREAD"
+#include "sl_power_manager.h"
 
 void sl_ot_init(void)
 {
   sl_ot_create_instance();
-  sl_ot_sleep_init();
+  sl_ot_sleep_init();  
+  
+  // TODO: Remove the Power Manager remove req when OT does not add an EM1 req at init
+  sl_power_manager_remove_em_requirement(SL_POWER_MANAGER_EM1);
```

Comment:
Just for general knowledge in case we play with the inits in the future, does it explicitly need to be before cli init or as long as it is after the sleep it is fine? Basically, at what point does it become a problem if we haven't called this?